Upgrade the codebase after upgrading ansible, and update the packages

autn opened the pull request:
Hi, 

This PR is used for upgrading the codebase (after upgrading to the new Ansible), and updating the dependent packages.

Please help me to review and merge. Thanks!

YPCrumble commented on roles/db/tasks/main.yml:
```diff
@@ -63,6 +63,16 @@
+  become_user: postgres
+  community.postgresql.postgresql_privs:
+    db: "{{ db_name }}"
+    privs: ALL
```
@autn what error does this step solve? Is there a related stackoverflow answer or GitHub you could link me to?

What I'm wondering in particular is whether we should provide `ALL` privileges or whether a more limited set of privileges is appropriate

YPCrumble replied:
One thing I'm wondering here is whether we should use `USAGE` instead of `ALL`...I'm not sure and I can't find documentation explaining the difference. Would `USAGE` work for the repo?

autn replied:
@YPCrumble I'll try to re-build a server to make sure it will work or not!

YPCrumble reviewed (commented):
@autn this is great, thank you! Just one small question below.

YPCrumble commented:
@autn I think this line is the cause of the tests failing: https://share.zight.com/9ZuLmDRX

autn commented:
> @autn I think this line is the cause of the tests failing: https://share.zight.com/9ZuLmDRX

@YPCrumble I don't think so, because the other PR has the same: https://github.com/jcalazan/ansible-django-stack/actions/runs/6652180232/job/18084436939?pr=198

I think the issue is within Molecule default > destroy task: `"Error connecting: Error while fetching server API version: HTTPConnection.request() got an unexpected keyword argument 'chunked'"`

YPCrumble commented:
@autn any idea on how to get the tests passing?

autn commented:
@YPCrumble Looks like it's an issue with upgrading dependencies. I'm working to debug one by one!

autn commented:
@YPCrumble I fixed the tests by downgrading the molecule. For schema public, I changed to `USAGE`. When I try to setup new server, if I have the error, I'll back to it!

YPCrumble commented:
@autn I'm OK with using `ALL` instead of `USAGE` - I think that should be fine to keep because we're also granting all privileges on the database!
